__tests__/healthcareProfessional.test.ts

```diff
@@ -0,0 +1,199 @@
+import { expect } from '@jest/globals'
+import resolvers from '../src/resolvers'
+import loadSchema from '../src/schema'
+import request from 'supertest'
+import { ApolloServer } from '@apollo/server'
+import { startStandaloneServer } from '@apollo/server/standalone'
+import { initiatilizeFirebaseInstance } from '../src/firebaseDb'
+
+const facilityIds = [] as string[]
+
+const facilityQueryData = {
```

Comment:
Ideally, we'd like a factory to create a Facility for us so this wouldn't be needed.